### PR TITLE
enh(doc): external PR documentation

### PR DESCRIPTION
**Pull request by sc979**
# Pull Request Template

## Description

Better manage the contributors PR and integrate it to our workflow
Enhance the github documentation on how to contribute.
Add specific tags -> 
image

**Fixes** # (external PR integration)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [x] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [x] 18.10.x
- [x] 19.04.x
- [x] 19.10.x
- [x] 20.04.x (master)

#### Community contributors & Centreon team

- [x] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [x] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
...
+The pull request must comply with certain requirements which are set out in the following template
+[PULL_REQUEST_TEMPLATE.md](.github/PULL_REQUEST_TEMPLATE.md)
 
-> **Notice**: Any pull request that does not respect those requirements will be legitimately rejected !
+> **Notice**: A pull request which contain more than the described modifications or contain more than one issue will
+be rejected. Please take care of the description and open one pull request per issue
+
+If the PR matches the expected requirements, it will be added to a refinement session with the dev team and the
+product owner.
+If every is clear enough, it will integrate the dev processes and merged if successfully pass our CI's acceptance tests.
```
Comment (cgagnaire) [inline CONTRIBUTING.md]:
```suggestion
If everything is clear enough, the PR will integrate the dev processes and will be merged if it successfully passes our CI's acceptance tests
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
...
+Then our Quality Assurance team will test it, again, to avoid any regression before being able to release it.
+
+If the dev team need more details, they'll contact you on the pull request. Please stay tuned.
```
Comment (cgagnaire) [inline CONTRIBUTING.md]:
```suggestion
If the dev team needs more details, they will contact you on the pull request. Please stay tuned.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
...
+> **Notice**: We inspired ourselves with another open source project we use, to better communicate on your work :
+Visual Studio Code.
+
+
+Here are the labels we may add to your work.
+
+| Label                             | Reason |
+| :-----                            | :------ |
+| ```pr/external```                 | The first badge |
+| ```status/accepted```             | The dev team have approved your modifications |
```
Comment (cgagnaire) [inline CONTRIBUTING.md]:
```suggestion
| ```status/accepted```             | The dev team has approved your modifications |
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
...
+| ```status/implemented```          | The PR have been added or may already be released in next version |
+| ```status/duplicate```            | Another PR is already open |
+| ```status/invalid```              | The requirements are not met |
+| ```status/in-backlog```           | The dev team have groomed your proposition and will work on it soon |
```
Comment (cgagnaire) [inline CONTRIBUTING.md]:
```suggestion
| ```status/in-backlog```           | The dev team has groomed your proposition and will work on it soon |
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
...
+| ```status/could-not-reproduce```  | The dev was unable to reproduce the use-case and may need more information |
+| ```status/needs-attention```      | This PR is on hold. The reason is specified in the issue |
+| ```status/needs-merge```          | A dev need to merge your work |
```
Comment (cgagnaire) [inline CONTRIBUTING.md]:
```suggestion
| ```status/needs-merge```          | A dev needs to merge your work |
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
 
 You have been working on Centreon base code and want to submit it to us. Well... Again you are more than welcome and thank you in advance ! :clap:
 
-The pull request should respect the some requirements that can found in the following template [PULL_REQUEST_TEMPLATE.md](.github/PULL_REQUEST_TEMPLATE.md)
+The pull request must comply with certain requirements which are set out in the following template
+[PULL_REQUEST_TEMPLATE.md](.github/PULL_REQUEST_TEMPLATE.md)
 
-> **Notice**: Any pull request that does not respect those requirements will be legitimately rejected !
+> **Notice**: A pull request which contain more than the described modifications or contain more than one issue will
+be rejected. Please take care of the description and open one pull request per issue
```
Comment (cgagnaire) [inline CONTRIBUTING.md]:
```suggestion
be rejected. Please take care of the description and open one pull request per issue.
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
 
 You have been working on Centreon base code and want to submit it to us. Well... Again you are more than welcome and thank you in advance ! :clap:
 
-The pull request should respect the some requirements that can found in the following template [PULL_REQUEST_TEMPLATE.md](.github/PULL_REQUEST_TEMPLATE.md)
+The pull request must comply with certain requirements which are set out in the following template
+[PULL_REQUEST_TEMPLATE.md](.github/PULL_REQUEST_TEMPLATE.md)
 
-> **Notice**: Any pull request that does not respect those requirements will be legitimately rejected !
+> **Notice**: A pull request which contain more than the described modifications or contain more than one issue will
```
Comment (cgagnaire) [inline CONTRIBUTING.md]:
```suggestion
> **Notice**: A pull request which contains more than the described modifications or contains more than one issue will
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
...
+| ```status/accepted```             | The dev team has approved your modifications |
+| ```status/implemented```          | The PR have been added or may already be released in next version |
+| ```status/duplicate```            | Another PR is already open |
+| ```status/invalid```              | The requirements are not met |
+| ```status/in-backlog```           | The dev team has groomed your proposition and will work on it soon |
+| ```status/could-not-reproduce```  | The dev was unable to reproduce the use-case and may need more information |
+| ```status/needs-attention```      | This PR is on hold. The reason is specified in the issue |
+| ```status/needs-merge```          | A dev needs to merge your work |
+| ```status/too-dangerous```        | The modification seems to impact too much features or may introduce side effects |
+| ```status/multiple-issues```      | The modifications impact more than one issue in a single PR |
```
Comment (sc979) [inline CONTRIBUTING.md]:
```suggestion
| ```status/multiple-issues```      | The modifications contains more than one issue in a single PR |
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
...
+| ```status/implemented```          | The PR have been added or may already be released in next version |
+| ```status/duplicate```            | Another PR is already open |
+| ```status/invalid```              | The requirements are not met |
+| ```status/in-backlog```           | The dev team has groomed your proposition and will work on it soon |
+| ```status/could-not-reproduce```  | The dev was unable to reproduce the use-case and may need more information |
+| ```status/needs-attention```      | This PR is on hold. The reason is specified in the issue |
+| ```status/needs-merge```          | A dev needs to merge your work |
+| ```status/too-dangerous```        | The modification seems to impact too much features or may introduce side effects |
+| ```status/multiple-issues```      | The modifications contains more than one issue in a single PR |
+| ```status/out-of-scope```         | The issue is out of the described scope |
```
Comment (sc979) [inline CONTRIBUTING.md]:
```suggestion
| ```status/out-of-scope```         | The modifications are out of the described scope |
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -128,12 +166,3 @@ For other languages, coding style rules are defined in Centreon GitHub repositor
 
 If you want to visualize and suggest modification to the documentation through a pull request
 you can check the **HOW TO** build the documentation section [here](doc/README.md)
```
Comment (sc979) [inline CONTRIBUTING.md]:
```suggestion
you can check the **HOW TO** build the documentation section [here](doc/README.md)

```

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
...
+
+If the PR matches the expected requirements, it will be added to a refinement session with the dev team and the
+product owner.
+If everything is clear enough, the PR will integrate the dev processes and will be merged if it successfully passes our CI's acceptance tests
+Then our Quality Assurance team will test it, again, to avoid any regression before being able to release it.
+
+If the dev team needs more details, they will contact you on the pull request. Please stay tuned.
+
+> **Warning**: Any pull request that does not respect those requirements will be legitimately rejected ! In addition,
+if you are asked to do so, you must help us understand your changes or problem, and respond to us within 8 days.
```
Comment (sc979) [inline CONTRIBUTING.md]:
```suggestion
if you are asked to do so, you must help us understand your changes or behavior, and respond to us within 8 days.
```

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
...
+If the dev team needs more details, they will contact you on the pull request. Please stay tuned.
+
+> **Warning**: Any pull request that does not respect those requirements will be legitimately rejected ! In addition,
+if you are asked to do so, you must help us understand your changes or behavior, and respond to us within 8 days.
+
+> **Notice**: We inspired ourselves with another open source project we use, to better communicate on your work :
+Visual Studio Code.
+
+
+Here are the labels we may add to your work.
```
Comment (sc979) [inline CONTRIBUTING.md]:
```suggestion
Here are the labels and descriptions we may add to your work.
```

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
...
+Visual Studio Code.
+
+
+Here are the labels and descriptions we may add to your work.
+
+| Label                             | Reason |
+| :-----                            | :------ |
+| ```pr/external```                 | The first badge |
+| ```status/accepted```             | The dev team has approved your modifications |
+| ```status/implemented```          | The PR have been added or may already be released in next version |
```
Comment (cgagnaire) [inline CONTRIBUTING.md]:
```suggestion
| ```status/implemented```          | The PR has been added or may have already been released in next version |
```

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
...
+| ```status/duplicate```            | Another PR is already open |
+| ```status/invalid```              | The requirements are not met |
+| ```status/in-backlog```           | The dev team has groomed your proposition and will work on it soon |
+| ```status/could-not-reproduce```  | The dev was unable to reproduce the use-case and may need more information |
+| ```status/needs-attention```      | This PR is on hold. The reason is specified in the issue |
+| ```status/needs-merge```          | A dev needs to merge your work |
+| ```status/too-dangerous```        | The modification seems to impact too much features or may introduce side effects |
+| ```status/multiple-issues```      | The modifications contains more than one issue in a single PR |
```
Comment (cgagnaire) [inline CONTRIBUTING.md]:
```suggestion
| ```status/multiple-issues```      | The modifications contain more than one issue in a single PR |
```

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
...
 
-The pull request should respect the some requirements that can found in the following template [PULL_REQUEST_TEMPLATE.md](.github/PULL_REQUEST_TEMPLATE.md)
+The pull request must comply with certain requirements which are set out in the following template
+[PULL_REQUEST_TEMPLATE.md](.github/PULL_REQUEST_TEMPLATE.md)
 
-> **Notice**: Any pull request that does not respect those requirements will be legitimately rejected !
+> **Notice**: A pull request which contains more than the described modifications or contains more than one issue will
+be rejected. Please take care of the description and open one pull request per issue.
+
+If the PR matches the expected requirements, it will be added to a refinement session with the dev team and the
```
Comment (adr-mo) [inline CONTRIBUTING.md]: `dev team` -> `development team`

File: CONTRIBUTING.md [hunk 16]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
...
+product owner.
+If everything is clear enough, the PR will integrate the dev processes and will be merged if it successfully passes our CI's acceptance tests
```
Comment (adr-mo) [inline CONTRIBUTING.md]: `CI's` -> `Continous Integration's`

Comment (adr-mo) [inline CONTRIBUTING.md]: `the PR will integrate the dev processes` -> `the PR will integrate our development workflow`

File: CONTRIBUTING.md [hunk 17]
```diff
@@ -74,9 +74,44 @@ Any ideas, enhancements, feature requests are more than welcome. Feature request
...
+Then our Quality Assurance team will test it, again, to avoid any regression before being able to release it.
+
+If the dev team needs more details, they will contact you on the pull request. Please stay tuned.
```
Comment (adr-mo) [inline CONTRIBUTING.md]: dev team -> development team

**Pull request merged**